index.html

```diff
@@ -712,101 +712,79 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
-    <section id="messaging">
-      <h2>Messaging</h2>
+    <section id="notifications">
+      <h2>Notifications</h2>
       <section class="informative">
         <h3>Introduction</h3>
         <p>
-          This section defines when messages are emitted by a Fedora
-          implementation, the minimal set of data contained in those messages and how
-          those data are serialized. These messages may occur synchronously or asynchronously
-          with the API operations that cause them to be emitted. They are typically used to
-          support external integrations. The structure of these messages draws upon the
+          This section defines when notifications are made available by a Fedora
+          implementation, the minimal set of data contained in these notifications and how
+          the data are serialized. Notifications may be emitted synchronously or asynchronously
+          with the API operations that cause them to be emitted. These notifications are typically
+          used to support external integrations. The structure of these notifications draws upon the
           existing [[activitystreams-core]] and [[ldn]] specifications.
-          An implementation is free to choose from any messaging technology so long as
-          the messages conform to what is described in the following sections.
-        </p>
-      </section>
-
-      <section id="emitting-messages">
-        <h3>Emitting Messages</h3>
-        <p>
-          Any operation that causes the state of a resource to durably change MUST cause a
-          message to be emitted with a corresponding resource identifier.
-        </p>
-        <p>
-          A failed operation (HTTP 4xx or 5xx) MUST NOT emit any messages.
-        </p>
-        <p>
-          Messages MUST NOT be emitted until after any changes have been persisted to durable storage.
+          An implementation is free to choose from any transport technology so long as
+          the notifications conform to what is described in the following sections.
         </p>
         <p>
-          Repository start, stop and configuration change operations MAY cause a message to be emitted.
+          Implementers should be aware that some operations cause multiple resources to change.
+          In these cases, there will be a corresponding notification describing each of the changes.
```

Comment:
I understood @awoods 's comment to be referring to the question of ordering-- that the spec should prevent inferring the ordering of operations from the ordering of notifications. Now I think I must have misunderstood the comment.